### PR TITLE
[zh] Update content/zh-cn/docs/reference/access-authn-authz/admission…

### PR DESCRIPTION
**Pull request by Yanping-io**

### CODE REVIEW

File: content/zh-cn/docs/reference/access-authn-authz/admission-controllers.md [hunk 1]
```diff
@@ -1494,6 +1494,16 @@ You should enable this admission controller if you intend to make any use of Kub
 的自动化。强烈推荐为 Kubernetes 项目启用此准入控制器。
 如果你打算使用 Kubernetes 的 `ServiceAccount` 对象，你应启用这个准入控制器。
 
+<!--
+Regarding the annotation `kubernetes.io/enforce-mountable-secrets`: While the annotation's name suggests it only concerns the mounting of Secrets,
+its enforcement also extends to other ways Secrets are used in the context of a Pod.
+Therefore, it is crucial to ensure that all the referenced secrets are correctly specified in the ServiceAccount.
+-->
+关于注释 `kubernetes.io/enforce-mountable-secrets` ：尽管注释的名称表明它只涉及 Secrets 的挂载，
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/admission-controllers.md:1502]:
```suggestion
关于 `kubernetes.io/enforce-mountable-secrets` 注解：尽管注解的名称表明它只涉及 Secret 的挂载，
```

File: content/zh-cn/docs/reference/access-authn-authz/admission-controllers.md [hunk 2]
```diff
@@ -1494,6 +1494,16 @@ You should enable this admission controller if you intend to make any use of Kub
...
+但其执行范围也扩展到 Pod 上下文中 Secrets 的其他使用方式。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/admission-controllers.md:1503]:
```suggestion
但其执行范围也扩展到 Pod 上下文中 Secret 的其他使用方式。
```

File: content/zh-cn/docs/reference/access-authn-authz/admission-controllers.md [hunk 3]
```diff
@@ -1494,6 +1494,16 @@ You should enable this admission controller if you intend to make any use of Kub
...
+因此，确保所有引用的 Secrets 在 ServiceAccount 中被正确指定是至关重要的。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/admission-controllers.md:1504]:
```suggestion
因此，确保所有引用的 Secret 在 ServiceAccount 中被正确指定是至关重要的。
```

**Review by windsonsea — COMMENTED** [review]
/approve

Please trigger the CI again.

**Comment by windsonsea** [comment]
Run the following commands to trigger ci again:

```
git reset
git commit --amend --no-edit
git push -f
```

**Comment by Yanping-io** [comment]
> Run the following commands to trigger ci again:
> 
> ```
> git reset
> git commit --amend --no-edit
> git push -f
> ```

thank you !

**Pull request merged**